src/app/public/testing/phonefield/phone-field-fixture.spec.ts

```diff
@@ -0,0 +1,193 @@
+  SkyPhoneFieldNumberReturnFormat
+} from '@skyux/phone-field';
+
+import {
+  SkyPhoneFieldFixture
+} from './phone-field-fixture';
+
+import {
+  SkyPhoneFieldTestingModule
+} from './phone-field-testing.module';
+
+const COUNTRY_AU: SkyPhoneFieldCountry = {
+  name: 'Australia',
+  iso2: 'au',
+  dialCode: '+61'
+};
+const COUNTRY_US: SkyPhoneFieldCountry = {
+  name: 'United States',
+  iso2: 'us',
+  dialCode: '+1'
+};
+const DATA_SKY_ID = 'test-phone-field';
+const VALID_AU_NUMBER = '0212345678';
+const VALID_US_NUMBER = '8675555309';
+
+//#region Test component
+@Component({
+  selector: 'phone-field-test',
+  template: `
+  <form
+    class='phone-field-demo'
+    [formGroup]="phoneForm"
+  >
+    <sky-phone-field
+      data-sky-id="${DATA_SKY_ID}"
+      [allowExtensions]="allowExtensions"
+      [defaultCountry]="defaultCountry"
+      [returnFormat]="returnFormat"
+      [supportedCountryISOs]="supportedCountryISOs"
+      [(selectedCountry)]="selectedCountry"
+      (selectedCountryChange)="selectedCountryChange($event)"
+    >
+      <input
+        formControlName="phoneControl"
+        skyPhoneFieldInput
+        [skyPhoneFieldNoValidate]="noValidate"
+      >
+    </sky-phone-field>
+
+    <sky-status-indicator *ngIf="!phoneControl.valid"
+      descriptionType="none"
+      indicatorType="danger"
+    >
+      Enter a phone number matching the format for the selected country.
+    </sky-status-indicator>
+  </form>
+`
+})
+class PhoneFieldTestComponent implements OnInit {
+  public allowExtensions: boolean = true;
+  public defaultCountry: string;
+  public noValidate: boolean = false;
+  public returnFormat: SkyPhoneFieldNumberReturnFormat;
+  public selectedCountry: SkyPhoneFieldCountry;
+  public showInvalidDirective: boolean = false;
+  public supportedCountryISOs: string[];
+
+  public phoneControl: FormControl;
+  public phoneForm: FormGroup;
+
+  public selectedCountryChange(query: string) { }
+
+  public ngOnInit(): void {
+    this.phoneControl = new FormControl();
+    this.phoneForm = new FormGroup({
+      'phoneControl': this.phoneControl
+    });
+  }
+}
+//#endregion Test component
+
+fdescribe('PhoneField fixture', () => {
+  let fixture: ComponentFixture<PhoneFieldTestComponent>;
+  let testComponent: PhoneFieldTestComponent;
+  let phonefieldFixture: SkyPhoneFieldFixture;
+
+  beforeEach(async () => {
+    TestBed.configureTestingModule({
+      declarations: [
+        PhoneFieldTestComponent
+      ],
+      imports: [
+        FormsModule,
+        ReactiveFormsModule,
+        SkyPhoneFieldTestingModule,
+        SkyStatusIndicatorModule
+      ]
+    });
+
+    // create the fixture, waiting until it's stable since it selects a country on init
+    fixture = TestBed.createComponent(
+      PhoneFieldTestComponent
+    );
+    testComponent = fixture.componentInstance;
+    phonefieldFixture = new SkyPhoneFieldFixture(fixture, DATA_SKY_ID);
+  });
+
+  it('should allow exensions by default', async () => {
```

Comment:
Both of these allow extensions tests seem unnecessary to me. I stopped adding tests because I wasn't testing anything exposed in the fixture itself in this case. It seems like these two should be removed to me. 

That would leave just the two tests about selecting a country and verifying the model after valid inputs. If you feel like that covers things, or there are other test cases I should cover, please let me know.

Reply:
Agreed. This is testing the `SkyPhoneFieldComponent` and should already be covered.